Dependent libraries not included

Should use `api()`

Getting following error

```
FATAL EXCEPTION: main
                                                                                                    Process: dev.hossain.timeline, PID: 10559
                                                                                                    java.lang.NoClassDefFoundError: Failed resolution of: Lcom/squareup/moshi/Moshi$Builder;
                                                                                                    	at dev.hossain.timeline.Parser$parse$4.invokeSuspend(Parser.kt:50)
                                                                                                    	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
                                                                                                    	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
                                                                                                    	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:113)
                                                                                                    	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89)
                                                                                                    	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:589)
                                                                                                    	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:823)
                                                                                                    	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:720)
                                                                                                    	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:707)
                                                                                                    	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [androidx.compose.ui.platform.MotionDurationScaleImpl@f7fb9c, androidx.compose.runtime.BroadcastFrameClock@ee7d9a5, StandaloneCoroutine{Cancelling}@7a6217a, AndroidUiDispatcher@e4b9d2b]
                                                                                                    Caused by: java.lang.ClassNotFoundException: Didn't find class "com.squareup.moshi.Moshi$Builder" on path: DexPathList[[dex file "/data/data/dev.hossain.timeline/code_cache/.overlay/base.apk/classes5.dex", dex file "/data/data/dev.hossain.timeline/code_cache/.overlay/base.apk/classes7.dex", zip file "/data/app/~~lhJNPCS194v-g2fY3RTIZA==/dev.hossain.timeline-KJrGVTMWd9hKbXHqpLMBMg==/base.apk"],nativeLibraryDirectories=[/data/app/~~lhJNPCS194v-g2fY3RTIZA==/dev.hossain.timeline-KJrGVTMWd9hKbXHqpLMBMg==/lib/arm64, /data/app/~~lhJNPCS194v-g2fY3RTIZA==/dev.hossain.timeline-KJrGVTMWd9hKbXHqpLMBMg==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
                                                                                                    	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
                                                                                                    	at java.lang.ClassLoader.loadClass(ClassLoader.java:637)
                                                                                                    	at java.lang.ClassLoader.loadClass(ClassLoader.java:573)
```


and 


```
Process: dev.hossain.timeline, PID: 9726 java.lang.NoClassDefFoundError: Failed resolution of: Lokio/Okio; at dev.hossain.timeline.Parser$parse$4.invokeSuspend(Parser.kt:47) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101) at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:113) at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89) at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:589) at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:823) at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:720) at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:707) Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [androidx.compose.ui.platform.MotionDurationScaleImpl@e021f0f, androidx.compose.runtime.BroadcastFrameClock@f7fb9c, StandaloneCoroutine{Cancelling}@ee7d9a5, AndroidUiDispatcher@7a6217a] Caused by: java.lang.ClassNotFoundException: Didn't find class "okio.Okio" on path: DexPathList[[dex file "/data/data/dev.hossain.timeline/code_cache/.overlay/base.apk/classes5.dex", dex file "/data/data/dev.hossain.timeline/code_cache/.overlay/base.apk/classes7.dex", zip file "/data/app/~~lhJNPCS194v-g2fY3RTIZA==/dev.hossain.timeline-KJrGVTMWd9hKbXHqpLMBMg==/base.apk"],nativeLibraryDirectories=[/data/app/<span>lhJNPCS194v-g2fY3RTIZA==/dev.hossain.timeline-KJrGVTMWd9hKbXHqpLMBMg==/lib/arm64, /data/app/</span>lhJNPCS194v-g2fY3RTIZA==/dev.hossain.timeline-KJrGVTMWd9hKbXHqpLMBMg==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]] at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259) at java.lang.ClassLoader.loadClass(ClassLoader.java:637) at java.lang.ClassLoader.loadClass(ClassLoader.java:573) ... 9 more
```

--- a/lib/build.gradle.kts
+++ b/lib/build.gradle.kts
@@ -4,7 +4,7 @@ plugins {
 }
 
 group = "dev.hossain.timeline"
-version = "1.2-SNAPSHOT"
+version = "1.3-SNAPSHOT"
 
 repositories {
     mavenCentral()
@@ -19,8 +19,8 @@ val okioVersion = "3.9.1"
 
 dependencies {
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutinesVersion")
-    implementation("com.squareup.moshi:moshi-kotlin:$moshiVersion")
-    implementation("com.squareup.okio:okio:$okioVersion")
+    api("com.squareup.moshi:moshi-kotlin:$moshiVersion")
+    api("com.squareup.okio:okio:$okioVersion")
 
     testImplementation(kotlin("test"))
     testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:$coroutinesVersion")

